clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -713,32 +713,6 @@ public void testInterceptorConstructorClose(GroupProtocol groupProtocol) {
         }
     }
 
-    @ParameterizedTest
-    @EnumSource(GroupProtocol.class)
-    public void testInterceptorConstructorConfigurationWithExceptionShouldCloseRemainingInstances(GroupProtocol groupProtocol) {
```

Comment:
Why is this test removed?

Reply:
I think we should keep this test, but update it to use only a single MockConsumerInterceptor.